MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.Dark.xaml

```diff
@@ -20,6 +20,10 @@
     <SolidColorBrush x:Key="MaterialDesignDivider" Color="#1FFFFFFF" po:Freeze="True" /><!-- 12% -->
     <SolidColorBrush x:Key="MaterialDesignSelection" Color="#757575" po:Freeze="True" />
 
+    <SolidColorBrush x:Key="MaterialDesignToolButtonForeground" Color="#616161" po:Freeze="true" />
```

Comment:
I don't think we need this extra resource. Ideally we only use the `MaterialDesignToolForeground` and `MaterialDesignToolBackground` resources.

Reply:
That's true there isn't need to duplicate there resource while the base contains the same declaration.